src/common/pages/apiDocumentation/APIitems/account_history_apis.tsx

```diff
@@ -0,0 +1,47 @@
+
+import React from 'react';
+import { connect, useSelector } from 'react-redux';
```

Comment:
Couldn't we do single generic component with props for these APIitems?

Reply:
Yes i was also thing about it, but i thing the single file will become to lengthy but i will take some time , it should be like that because it is good for re-usability